src/main.rs

```diff
@@ -41,7 +41,11 @@ fn execute(args: &ArgMatches) -> Result<i32, error::FatalError> {
             config::get_release_config(&cargo_file, config::DOC_BRANCH).and_then(|f| f.as_str())
         })
         .unwrap_or("gh-pages");
-    let doc_commit_msg = config::get_release_config(&cargo_file, config::DOC_COMMIT_MESSAGE)
+    let doc_commit_msg = args.value_of("doc-commit-message")
+        .or_else(|| {
+            config::get_release_config(&cargo_file, config::DOC_COMMIT_MESSAGE)
+                .and_then(|f| f.as_str())
+        })
         .and_then(|f| f.as_str())
```

Comment:
Yes... it's breaking the build. Messed up breaking up the patch. I'll put a fix up when I have the container built again with your SUDO_CHECK fix.